common/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft/src/Default/NewtonSoftJsonSerializer.cs

```diff
@@ -231,8 +231,17 @@ internal class JsonVariantValue : VariantValue {
             /// <inheritdoc/>
             public override object ConvertTo(Type type) {
                 try {
-                    return Token.ToObject(type,
-                        JsonSerializer.CreateDefault(_serializer.Settings));
+                        if (type != typeof(List<string>)) {
+                            return Token.ToObject(type,
+                                JsonSerializer.CreateDefault(_serializer.Settings));
+                        }
+                        else {
+                            List<string> output = new List<string>();
```

Comment:
C# guidelines recommend using "var" when type is apparent (e.g. when using "new"). It would be great to reach agreement on this so that we can all be consistent :) https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#implicitly-typed-local-variables

Reply:
var please!

Reply:
changed